android/app-demo/src/main/java/io/projectnewm/demo/Navigation.kt

```diff
@@ -4,14 +4,14 @@ import androidx.compose.runtime.Composable
 import androidx.navigation.compose.NavHost
 import androidx.navigation.compose.composable
 import androidx.navigation.compose.rememberNavController
-import io.projectnewm.feature.example.ExampleScreen
+//import io.projectnewm.feature.example.ExampleScreen
```

Comment:
We are removing the example code on this [PR](https://github.com/projectNEWM/newm-mobile/pull/74)